examples/wolf_sheep_experimental/Readme.md

```diff
@@ -4,17 +4,20 @@ This is an experimental implementation of the Wolf-Sheep Predation Model, which
 
 ## Summary
 
-This model consists of three agent types: wolves, sheep, and grass. The wolves and the sheep wander around the grid at random. Wolves and sheep both expend energy moving around and replenish it by eating. Sheep eat grass, and wolves eat sheep if they end up on the same grid cell.
+## Summary
+
+This model consists of three agent types: wolves, sheep, and grass patches. The wolves and sheep wander around the grid at random, expending energy as they move. Sheep eat fully grown grass to replenish energy, while wolves eat sheep if they occupy the same cell. 
 
-If wolves and sheep have enough energy, they reproduce, creating a new wolf or sheep (in this simplified model, only one parent is needed for reproduction). The grass on each cell regrows at a constant rate. If any wolves and sheep run out of energy, they die.
+Both species reproduce asexually with a certain probability if they have sufficient energy, splitting their energy with offspring. The grass regrows after a set time if enabled in the model. Agents die if their energy depletes.
```

Comment:
Reproduce sexually, that implies two sheep or wolf are needed. Is this reflected in the model? And if so, do they have genders and do you need one of each, or is that simplified.

I would propose to keep the example not more complex than needed.